### DISCUSSION TITLE
GameMaker Studio 2

### DISCUSSION THREAD

**Discussion by ghost** [catppuccin/catppuccin#2209] [Port Requests]
### Will you make the port?

Possibly, if I get more time

### How can the suggested port be themed?

GameMaker Studio has had several custom themes developed for its IDE interface over the years, the most prominent examples being the [Dracula](https://github.com/dracula/gamemaker-studio) theme and the [Gruvbox](https://github.com/heygleeson/Gruvbox-GMTheme) theme.

GameMaker stores files for different themes in it's "Skins" folder (C:\ProgramData\GameMakerStudio2\Skins). It stores JSON data for colors and styles for the interface, interface icons, and other graphics. The above examples provide a good example of how a theme can be developed for GameMaker Studio 2. :)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sofycat** [comment]
hello hello!

i've started porting gamemaker in my free time!

Currently got most of frappe finished!
https://github.com/sofycat/gamemaker

**Comment by scarcekoi** [comment]
I am also working on one in my free time with the base of sofycat's:
https://github.com/scarcekoi/gamemaker
